app/views/clients/_enrollment_table.haml

```diff
@@ -78,6 +78,10 @@
             - title += ";<br /><b>Move-in Date:</b> No move-in date"
           %span{data: { toggle: :tooltip, title: title.html_safe, html: 'true' }}
             = e[:entry_date]
+          - is_residential = true # FIXME how do I get this
```

Comment:
`is_residential` equates to `calculated_project_type.in?(GrdaWarehouse::Hud::Project.RESIDENTIAL_PROJECT_TYPE_IDS)`

Reply:
I'd move this calculation to the client_extension so you can just show it if it's true, and non-residential projects will always be false.  (Preference complexity on the model over the view.)